execution/executor/src/lib.rs

```diff
@@ -192,7 +194,11 @@ where
 
         // 2. Verify that skipped transactions match what's already persisted (no fork):
         let num_txns_to_skip = num_committed_txns - first_txn_version;
-        info!("Skipping the first {} transactions.", num_txns_to_skip);
+
+        send_struct_log!(executor_log(LogEntry::Chunk).data(
+            logging::EVENT,
+            format!("Skipping the first {} transactions.", num_txns_to_skip)
```

Comment:
not the most effective structured log :P

would be better to 
event skipping transactions
count x